### PR TITLE
Add monitoring links for Bitnami Kafka manifests

### PR DESCRIPTION
**Pull request by matthewhughes-uw**
Add links to existing, parameterised Grafana dashboards that teams using these manifests can make use of

### CODE REVIEW

**Review by sbuliarca — APPROVED** [review]

File: bitnami/README.md [hunk 1]
```diff
@@ -45,3 +47,12 @@ pubsub
...
 * PR the changes
 * use the new base in `kubernetes-manifests`
+
+## Monitoring
+
+For clusters using these manfiests, there are Grafana dashboards available for:
+
+  - [A general
+    overview](https://grafana.dev.aws.uw.systems/d/qoF35R8Vk/bitnami-kafka-overview?from=now-24h&to=now&var-datasource=default&var-namespace=pubsub&var-topic=All&var-consumergroup=All&orgId=1&refresh=1m)
+  - [Topic and consumer group
```
Comment (sbuliarca) [inline bitnami/README.md:57]: This one has preselected the es-topic-indexer. I'd leave it to all.

File: bitnami/README.md [hunk 2]
```diff
@@ -45,3 +47,12 @@ pubsub
 * add more resources to be base if needed
 * PR the changes
 * use the new base in `kubernetes-manifests`
+
+## Monitoring
```
Comment (sbuliarca) [inline bitnami/README.md:51]: Maybe split those links by cluster, as they are avaible in dev, prod accross all providers aws, merit and gcp

File: bitnami/README.md (hunk 1 above)
Comment (matthewhughes-uw) [reply bitnami/README.md]: > This one has preselected the es-topic-indexer. I'd leave it to all or just take out any parameters from the link

Good spot: [7de2089](https://github.com/utilitywarehouse/kafka-manifests/pull/59/commits/7de20896d5cec8b96e257c343422a4e7b845fa92)

File: bitnami/README.md (hunk 2 above)
Comment (matthewhughes-uw) [reply bitnami/README.md:60]: > Maybe split those links by cluster, as they are avaible in dev, prod accross all providers aws, merit and gcp

[d0bd7ab](https://github.com/utilitywarehouse/kafka-manifests/pull/59/commits/d0bd7abbb03c70581b5d95a992e7b98e8a7be3cd)

File: bitnami/README.md [hunk 3]
```diff
@@ -1,47 +1,77 @@
...
+    	  -e BITNAMI_KAFKA_RELEASE=${BITNAMI_KAFKA_RELEASE} \
+    	  -e NS="pubsub" \
+    	  -e NAME="kafka-shared" \
+    	  -e KRAFT_CLUSTERID="k6fA260ZKAhRUMBN7aMApm" \
+    	  -e VALUES=${VALUES_TLS} \
+    	  --workdir=/opt/manifests \
+    	  --entrypoint=/bin/sh \
+    	  alpine/helm ./gen-yaml/gen
+
+  - execute the target - this will create a new dir containing the base
```
Comment (matthewhughes-uw) [inline bitnami/README.md:45]: whoops, accidentally formatted the entire file. I'll clean this up

File: bitnami/README.md [hunk 4]
```diff
@@ -1,47 +1,77 @@
+# Bitnami based manifests
+
 Creates a base from bitnami helm charts with minimal configuration
 
-`shared` contains kustomize bases that can be reused across namespaces
-`config` contains configuration files pass to the bitnami helm charts
-* currently contains the configuration for setting up a cluster with tls enabled
+`shared` contains kustomize bases that can be reused across namespaces `config`
```
Comment (sbuliarca) [inline bitnami/README.md:5]: Not sure why these diffs appear now

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**